### ISSUE TITLE
Update network file list in a client if new clients connect to the network

### ISSUE THREAD

**Issue by carlosgeos** [TheOnionBay/TORrents#32]
Alice will have an old outdated list of network files if many clients connect to the tracker after her.

The tracker should send an ls type of message on all known CID to inform the clients about the new files in the network

**Comment by carlosgeos** [comment]
An easy solution would be to use APScheduler to run a cron job and update the list of files in both client and tracker:

https://stackoverflow.com/questions/21214270/scheduling-a-function-to-run-every-hour-on-flask

```python
import atexit

from apscheduler.scheduler import Scheduler
from flask import Flask

app = Flask(__name__)

cron = Scheduler(daemon=True)
# Explicitly kick off the background thread
cron.start()

@cron.interval_schedule(hours=1)
def job_function():
    # Do your work here


# Shutdown your cron thread if the web process is stopped
atexit.register(lambda: cron.shutdown(wait=False))

if __name__ == '__main__':
    app.run()
```

**Comment by TheoVerhelst** [comment]
I implemented a simple solution in 4adb351ec2, @carlosgeos can you tell me if it has problems as opposed to your cron jobs solution?
